### ISSUE TITLE
DateTime columns get from database have Kind property Unspecified.

### ISSUE THREAD

**Issue by moxplod** [MoonStorm/Dapper.FastCRUD#39]
I setup a simple unit  test that adds a row to the database and the date that is added to the database has Kind property set when inserting it as Utc by default. As its added using DateTime.UtcNow.

When i get the same object back from Database using dapper / dapper fastcrud the same field has the correct datetime but the Kind property is Unspecified - when it should be Utc.

I realize that this could be something that maybe need to be put in dapper. Is there any overrides we can do to fastcrud library that can set the kind property to the datetimes retrieved?

**Comment by MoonStorm** [comment]
Not specific to any of the two libraries. It is a known annoyance. The SQL field has no space to store the time zone info. People recommend switching to ``datetimeoffset``.

**Comment by MoonStorm** [comment]
@moxplod how did you eventually solve this problem?

**Comment by moxplod** [comment]
I ended up not implementing anything at the Data access layer for this, As that would affect all the dates while performing the mapping to DTOs. Was'nt sure if the performance would be good when we do that. In our scenario - we had to have the kind property populated in a few situations. So its handled in a higher layer when we need to convert to User's timezone - it checks if the Kind property of the datetime passed in is unspecified - then sets it to Utc.
